src/main/java/com/stuypulse/robot/subsystems/Manager.java

```diff
@@ -131,6 +139,19 @@ private ArmState getMidReadyTrajectory() {
         }
     }
 
+    private ArmState getAutonMidReadyTrajectory() {
+        switch (gamePiece) {
+            case CONE_TIP_UP:
+                return Ready.Mid.kConeTipUpBack;
+
+            case CUBE:
+                return scoreSide == ScoreSide.FRONT ? Ready.Mid.kCubeFront : Ready.Mid.kAutonCubeBack;
```

Comment:
Should this be in the auton itself since we already seperate the scoring trajectories?